backend/routes/runbooks.routes.js

```
@@ -0,0 +1,568 @@
+module.exports = router;
+
+// ─── Helpers ──────────────────────────────────────────────────────────────────
+
+/**
+ * Validate the common fields shared by POST and PUT.
+ * Returns an error string or null if valid.
+ *
+ * @param {Object} body
+ * @returns {string|null}
+ */
+function validateRunbookBody(body) {
```

Biome (2.4.4)

[error] Shouldn't redeclare 'validateRunbookBody'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)